src/main/kotlin/org/digma/intellij/plugin/ui/common/Panels.kt

```diff
@@ -63,16 +64,19 @@ private fun getNoInfoMessage(model: PanelModel): String {
 
 fun createTopPanel(project: Project, model: PanelModel): DigmaResettablePanel {
 
-    val scopeLine = scopeLine({ model.getScope() }, { model.getScopeTooltip() }, ScopeLineIconProducer(model))
-    scopeLine.isOpaque = false
+    var scopeLine: DialogPanel? = null
```

Comment:
is it desired feature? that scope line might be nullable?

Reply:
yes this is desirable scenario, because we should avoid SummaryModel of using this part of code.
on next line we are checking if this model is InsightsModel or ErrorsModel:
`    var scopeLine: DialogPanel? = null
    if (model is InsightsModel || model is ErrorsModel) {
        scopeLine = scopeLine({ model.getScope() }, { model.getScopeTooltip() }, ScopeLineIconProducer(model))
        scopeLine.isOpaque = false
    }`